web/modules/custom/ai_screening/src/Drush/Commands/Commands.php

```diff
@@ -117,15 +117,15 @@ public function accessCheck(
   /**
    * Pre command hook for pm:install.
    */
-  #[CLI\Hook(HookManager::PRE_COMMAND_HOOK, target: '*')]
+  #[CLI\Hook(HookManager::PRE_COMMAND_HOOK, target: 'pm:install')]
```

Comment:
What about the target `config:export`? Isn't that the main reason for implementing this hook?

Reply:
No pm-install with the original reason. Because when enabling modules drupal runs through some config stuff and updates it. When enabling through drush the user is anonymous and lang code is "Site language" which is da, but we want to enable modules as user 1 who is "en" so config system doesn't want to change all/some config to da

Reply:
I Changed to * due to some translation:export that acted up, but i don't think it made any difference. I just left the * part because i actually think it made some sense to be running drush commands as user one, but apparently it messes with user-login command, so i just switched it back